proto/buf.yaml

```diff
@@ -0,0 +1,10 @@
+version: v1
+name: buf.build/streamingfast/substreams-sink-database
```

Comment:
Wondering if it's the good package name `substreams-sink-database`, I personally like the fact that we have `-change` suffix, it gives a little help to get that it's about the changes and not sinking to any database (helps just a bit).

Also, do you think we should align GitHub repo name and buf protobuf package name?

Reply:
@abourget Might also have some opinions here.

Reply:
Do you want to rename the rust crate too?

Reply:
That would align everything indeed.

Reply:
That will be a breaking change probably, I need to check what we put as the Rust crate

Reply:
Aw man, the proto package is named `sf.substreams.sink.database.v1`. Would we rename that too? Seems like we are opening a can of worms here :-P

Reply:
Also, is it `database_change` or `database_changes`? The message we are emitting is pluralized ...

Reply:
Haha :) So .... in all that, we have also https://github.com/streamingfast/substreams-entity-change which is more "tailored" made to `graph-node` but which also have normalization issues.

With all those changes in those 2 packages, maybe we should actually do https://github.com/streamingfast/substreams-sink-postgres/issues/14. This would create a new version that would be incompatible anyway. 

For which we could normalize fully and also be built to be use by `graph-node`.